Formula/g/gorilla-cli.rb

```diff
@@ -18,6 +18,9 @@ class GorillaCli < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c3909a9070a0bc65842f887b937bb8f8613b5c40e4e4de86e8cd1ef4e82a41c2"
   end
 
+  # upstream bug report, https://github.com/gorilla-llm/gorilla-cli/issues/64
+  disable! date: "2024-09-12", because: "cli does not work due to `LoadingServer is unreachable`"
```

Comment:
Disabling for not responding to an issue after three weeks seems a bit much.

Reply:
yeah, just throw here for discussion.

the formula is not working atm at all. (I was thinking to deprecate in the first place)

Reply:
Short of disabling the formula, the test could be weakened to only check whether the command runs, even when the API is down.

Reply:
#192048 allows the test to pass for now, issue is an expired SSL cert